promscale/tobs/tobs.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/recommendations/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The observability stack (tobs), is a pre-packaged distribution of observability
+tools and dashboard interfaces which can be installed on any existing Kubernetes
```

Comment:
dashboards would imply that Tobs installs dashboards. I guess it is better to be technically accurate by stating that Tobs deploys dashboard interfaces(Grafana, Alert manager).

Reply:
Yes, Tobs installs these dashboards. I haven't seen dashboards suffixed with interfaces in Grafana's entire documentation while referencing dashboards or other docs. If we use interfaces it adds confusion to the users.

Reply:
Grafana or other docs do not use interface because it is their dashboard that they are talking about in their docs. Tobs does not have a dashboard of its own right? It is the dashboard interface that it installs.

Reply:
From my readings and findings across different OSS projects. I haven't come across the phrase dashboard interface(s) this adds processing overhead to the audience. Having it as just `dashboards` does the job. 

And there is no significant value difference for adding `dashboards` as `dashboard interfaces`. But the otherwise is definitely confusing.